### PR TITLE
[DiscordRPC] Add storefront/ Fix songs with no catalog ID

### PR DESCRIPTION
**Pull request by yazninja**

### CODE REVIEW

File: src/main/plugins/discordrpc.ts [hunk 1]
```diff
@@ -102,7 +102,15 @@ export default class DiscordRichPresence {
             return;
         }
         const listenURL = `https://cider.sh/p?s&id=${attributes.playParams.id}` // cider://play/s/[id] (for song)
-        const AMWebURL = `https://music.apple.com/us/song/${attributes.playParams.catalogId}`
+        let AMWebURL = '';
+        if(attributes.playParams.catalogId  == null){
+            AMWebURL = `https://music.apple.com/${attributes.storefrontId}/song/${attributes.playParams.id}`
```
Comment (quacksire) [inline src/main/plugins/discordrpc.ts:107]: seems unnecessary for the conditional except the console.log statement

**Comment by cryptofyre** [comment]
amongus

**Review by quacksire — CHANGES_REQUESTED** [review]
seems unnecessary for the conditional except the console.log statement

File: src/main/plugins/discordrpc.ts [hunk 2]
```diff
@@ -102,7 +102,15 @@ export default class DiscordRichPresence {
...
+            console.log("No Catalog ID");
+        }
+        else {
+            AMWebURL = `https://music.apple.com/${attributes.storefrontId}/song/${attributes.playParams.catalogId}`
```
Comment (quacksire) [inline src/main/plugins/discordrpc.ts:111]: seems unnecessary for the conditional except the console.log statement

File: src/main/plugins/discordrpc.ts (hunk 1 above)
Comment (yazninja) [reply src/main/plugins/discordrpc.ts:107]: if catalogID is null then use id

**Pull request merged**